### DISCUSSION TITLE
Suggestion: "Live" filtering box in place of "Search"

### DISCUSSION THREAD

**Discussion by Coldblackice** [streamlink/streamlink-twitch-gui#891] [General]
Would it be possible to add functionality (or the option) that, when viewing either of the "Followed" sub-tabs, (Followed) Streams + (Followed) Channels, the "Search" box instead acts as a live-filter box? Typing in the box would then filter out (temporarily) streams/channels that don't match the filter terms.

This would provide a fast and easy way of locating a particular stream/channel instead of search through a grid or having to type out and initiate a full search. Twitch-wide search. Likewise, sometimes I forget a channel name if there is unusual spelling or multiple words.

In any case, this is a sleek, phenomenally useful app. The additional functionality and customization options (even at a per-channel level!) are fantastic and well thought out. Awesome job!

**Comment by bastimeyer** [comment]
Please create a feature request thread on the issue tracker. The discussion forum is not the right place for this.
I can see locally filtering streams/channels/games via keywords being useful while typing a search query before submitting it.
